lib_collections/templates/lib_collections/exhibit_page.html

```diff
@@ -499,12 +499,18 @@ <h2>Exhibit Publications &amp; Documents</h2>
     <div class="rightside-mod col-xs-12 col-sm-6 col-md-12">
       <h2>Exhibit Details</h2>
       <p>
-        {% if self.exhibit_open_date or self.exhibit_close_date %}
-          <strong>
+        {% if self.space_type %}
+          {{ self.space_type }} Exhibit<br/>
+        {% endif %}
+
+        {% if self.is_online_exhibit %}
+            From {% if self.exhibit_open_date %}{{ self.exhibit_open_date }}{% endif %}
+            <br/>
+        {% else %}
             {% if self.exhibit_open_date %}{{ self.exhibit_open_date }}{% endif %}
             {% if self.exhibit_open_date and self.exhibit_close_date %}&ndash;{% endif %}
             {% if self.exhibit_close_date %}{{ self.exhibit_close_date }}{% endif %}
-          </strong><br>
+            <br/>
```

Comment:
I think I'm understanding this right; let me know if my change works as you are describing.
I originally put them in the clauses because I was thinking "what if there is no data" but that is probably not a thing that would happen in this case.

Reply:
The `else` clause executes if there isn't any data, so it's still being injected either way. At least, that's how I'm reading it.